__tests__/balance.test.js

```diff
@@ -1,10 +1,20 @@
 import TestUtils from './test-utils';
 
+const walletId = 'stub_balance';
+
 describe('balance api', () => {
+  beforeAll(async () => {
+    await TestUtils.startWallet({ walletId, preCalculatedAddresses: TestUtils.addresses });
```

Comment:
I didn't use the precalculated addresses where the address generation process is tested. Do you think it would be better to have this explicitly commented on the tests?